doc/guides/timers-in-node.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Timers in node.js
+layout: docs.hbs
+---
+
+# Timers in node.js and beyond
+
+Timers are a collection of global functions in node.js that allow you to 
+execute code after a set period of time. To fully understand when timer 
+functions will be executed, it's a good idea to read up on the the node 
+[Event Loop](https://nodesource.com/blog/understanding-the-nodejs-event-loop/).
+
+## Code Time Machine
+
+The node API provides several ways to schedule your code to execute at 
+some point after the present moment.
+
+### "When I say so" Execution ~ *setTimeout()*
+
+Using `setTimeout()`, you can schedule code to execute after a designated 
+amount of milliseconds. You may already be familiar with this function, as 
+it is a part of V8 and thus part of the browser JavaScript API.
+
+`setTimeout()` accepts the code to execute as its first argument and the 
+millisecond delay defined as a number literal as the second argument. Here 
+is an example of that:
```

Comment:
As noted [in the docs](https://nodejs.org/dist/latest-v6.x/docs/api/timers.html#timers_settimeout_callback_delay_arg), you can also pass any number of arguments though to the function being called via `setTimeout()`, beyond the 2nd argument.